### PR TITLE
complete fetp-22

### PR DESCRIPTION
**Pull request by Akhilesh1004**
I learn how to use nginx as web server and how the vue.js project work, more over, I learn more about how to use github

### CODE REVIEW

File: Akhilesh1004/report-fetp-22.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# FETP-22 Report
+
+## INFORMATION
+**GitHub ID**: `<your-github-id>`
```
Comment (imyungchu) [inline Akhilesh1004/report-fetp-22.md]: 這裡沒有改到 GitHub ID

File: Akhilesh1004/report-fetp-22.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+**ANSWER:** Staging Environment, Testing Environment
+
+---
+
+> **QUESTION:** 可以注意到在 Server 上有兩個地方都有 `routingapp` 的專案： `~/ansible/routingapp` 與 `/var/www/routingapp`。
+>
+> 請問您認為以上兩個目錄，誰更適合作為 Production，誰適合作為 Developing？
+
+**ANSWER:** ~/ansible/routingapp for Developing Environment and /var/www/routingapp for Developing Environment
```
Comment (imyungchu) [inline Akhilesh1004/report-fetp-22.md]: Why both of two directions are for Developing Environment?

**Review by imyungchu — COMMENTED** [review]
Finishing Review

File: Akhilesh1004/report-fetp-22.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+> **QUESTION:** 在這邊我們使用了加密演算法 ED25519。除了 ED25519 以外，還有一個很有名的加密演算法，叫什麼名字？
+
+**ANSWER:** RSA(Rivest-Shamir-Adleman)
+
+---
+
+> (optional) **QUESTION:** 這個有名的加密演算法有什麼問題，用了什麼方式降低了這個問題的風險？
+
+**ANSWER:** The problem with the RSA encryption algorithm is the factorization of large numbers, which may be broken by future computing power. To reduce risk, increase the RSA key length or switch to other more secure algorithms such as ED25519 or ECC.
```
Comment (Tangtang0910) [inline Akhilesh1004/report-fetp-22.md:22]: 這邊解釋得清楚好懂，很棒！

**Review by Tangtang0910 — APPROVED** [review]
review finished

File: Akhilesh1004/report-fetp-22.md (hunk 2 above)
Comment (kdotwei) [reply Akhilesh1004/report-fetp-22.md]: @imyungchu Please review and approve this PR again. Thanks!

**Comment by kdotwei** [comment]
@imyungchu You haven't approved this pull request; please check it.

**Review by imyungchu — APPROVED** [review]
Approve!

**Pull request merged**